src/rak.js

```diff
@@ -1,13 +1,17 @@
 const { EventEmitter } = require('events')
 const ConnWorker = require('./rakWorker')
 const { waitFor } = require('./datatypes/util')
+const { defaultOptions } = require('./options')
 
 let Client, Server, PacketPriority, EncapsulatedPacket, PacketReliability, Reliability
 
 module.exports = nativeRaknet => {
   if (nativeRaknet) {
     try {
-      ({ Client, Server, PacketPriority, PacketReliability } = require('raknet-native'))
+	  if (defaultOptions.useRustRaknet)
```

Comment:
If you use the default options here the user can't change it. Instead of passing the "useNativeRaknet" option to rak.js that can instead be deprecated in favor of a new more specific raknet backend option. Setting it as default will also allow CI to run on this. 

If (backend == "...") require(...)

Reply:
Not resolved

Reply:
I would like to have the new bindings backend as a user configuration, not as a default for now, so that there can be a transition time in the future to verify stability. I don't know what about Not resolved

Reply:
> If you use the default options here the user can't change it. Instead of passing the "useNativeRaknet" option to rak.js that can instead be deprecated in favor of a new more specific raknet backend option. Setting it as default will also allow CI to run on this.

If (backend == "...") require(...)